api/v1alpha1/mysqlaaduser_types.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
```

Comment:
probably remove all these “helper comments”

Reply:
Removed the `EDIT THIS FILE!` one but left the `json tags` note as it seemed helpful (although a bit obvious)